### ISSUE TITLE
[XamarinInspector-MAC]: Usability: VoiceOver: Once user is expanding or collapsing the UI tree from left pane of the Xamarin Inspector, the VoiceOver announces the same restults for infinitly  many times, as shown in the attachments.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#297]
###Steps to Reproduce

###Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio, Turn On the VoiceOver.
2. Navigate to Left pane (Tree View pane) of the inspector window.
3. Navigate through these tree items and expand/collapse  these UI trees using VO keys (the Control and Option keys) + space bar.
4. Observe the VoiceOver behavior once UI trees are being collapsed.
### Expected Behavior
The VoiceOver should announce only collapsed/Expanded UI tree, on which VoiceOver focus is present.
It should not announce all the trees which are expanded/collapsed in the below level.
### Actual Behavior
Once user is expanding or collapsing the UI tree from left pane of the Xamarin Inspector, the VoiceOver announces the same restults for infinitly  many times, as shown in the attachments.
### User Impact
Announcing the same results for the infinitly many time, it becomes very annoying for the screen-reader user.

[VO3 Repeatitive announcement.zip](https://github.com/Microsoft/workbooks/files/2000149/VO3.Repeatitive.announcement.zip)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
Accessibility
A11yUsable
Xamarin-Inspector-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
